www/Contributing.md

```diff
@@ -7,7 +7,10 @@ JNA contains work from many developers. You're encouraged to contribute to both
 - Fork the code from [github.com/java-native-access/jna](https://github.com/java-native-access/jna)
 - Check out the code with `git clone git@github.com:username/jna.git`
 - Ensure you can build the project with `ant dist test test-platform`
-- Make your code changes, write tests, build
+- Make your code changes, write tests, build. New code should follow the
+  conventions observable in the surrounding code/the codebase. Existing code
```

Comment:
Could you please explicitly state the conventions you want, for example "Traditional Bean Java Naming conventions". Otherwise you're asking for an undefined Standard, as there is no declared linter, nor linting test within the Testing Pipeline.

Reply:
@Crain-32 See #1473. Would appreciate any input you have there.